experimental/report/testdata/i18n.yaml.color.txt

```diff
@@ -1,18 +1,18 @@
 ⟨b.red⟩error: emoji, CJK, bidi⟨reset⟩
 ⟨blu⟩  --> foo.proto:5:9
-⟨blu⟩   | 
+   | 
```

Comment:
Is this part of the diff expected? Also, I notice that it doesn't (and didn't in previous commit) have a `(reset)` after (I think this is related/due to something I mentioned in another comment).

Reply:
Yes, this is a consequence of a comment you made about a second color code being unnecessary. And yes, the comment you made about reset and sidebars causes a reset to be added where you suggest.